Cargo.toml

```diff
@@ -71,17 +71,15 @@ swc_node_comments   = { version = "=0.23.0" }
 
 [profile.dev]
 codegen-units = 16
-debug         = 2       # debug build will cause runtime panic if codegen-unints is default
+debug         = 2    # debug build will cause runtime panic if codegen-unints is default
 incremental   = true
-panic         = "abort"
```

Comment:
55.9 MB -> 63.8 MB is a big degradation 😂, can you explain the benefits in more detail?

Reply:
`panic = abort` aborts the program whenever a panic occurs. Only with `panic = unwind` enabled can we catch the panic and add help message to the original panic thrown by the swc. See: https://doc.rust-lang.org/cargo/reference/profiles.html#panic

Reply:
Get, it should be helpful for error diagnosis, we can reduce the binary size in other ways